.python-version

```diff
@@ -0,0 +1 @@
+3.7.5
```

Comment:
it's kinda weird that you have this in the root directory, but the Pipfiles are in the s3migrate dir (but not a huge deal)